src/storybook/stand-alone-documentaion/spacing/spacing-sizes/spacing-component/spacing-component.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import cx from "classnames";
```

Comment:
this is the old spacing examples components with some changes for fitting to Evigeny design changes. I would do some more refactor (class names for example) but i though it will be overkill and we have a lot of work on the storybook already